src/resources/InsightPanelConfigs/InsightPanelConfig.ts

```diff
@@ -32,4 +33,10 @@ export default class InsightPanelConfig extends Resource {
 
         return this.api.put<InsightPanelConfigModel>(`${InsightPanelConfig.baseUrl}/${id}`, body);
     }
+
+    duplicate(duplicationParams: InsightPanelConfigDuplicateParams) {
+        const {id, name} = duplicationParams;
+
+        return this.api.post<InsightPanelConfigModel>(`${InsightPanelConfig.baseUrl}/${id}`, name);
```

Comment:
Is this endpoint documented in Swagger? I looked [here](https://platform.cloud.coveo.com/docs?urls.primaryName=Customer%20Service#/) but didn't see it.

Reply:
not currently but we will add it shortly